lib/adapter/external/manga_kakalot.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Ewbscp.Adapter.External.MangaKakalot do
+  use Tesla
+
+  plug(Tesla.Middleware.BaseUrl, "https://ww7.mangakakalot.tv")
+  plug(Tesla.Middleware.Headers, [{"authorization", "token xyz"}])
```

Comment:
Are we going to use this header? 🤔

Reply:
The BaseUrl one, yes. The others, no. I forgot to take this off.